test/helpers/test.js

```diff
@@ -149,6 +149,13 @@
         },
     };
 
+    // Log the name of the currently running test when it starts. Uses console.log rather than
+    // $.console.log so that the message is printed even after the $.console is diverted (see below).
+    // Uncomment this code to log which test is currently running. Useful for debugging.
```

Comment:
It says "uncomment this code", yet it's already uncommented. I think it's fine to have it always do it, so maybe we don't need that line of comment?

Reply:
If you're good with always printing these message, I agree we can remove the line about (un)commenting the code.

Reply:
Yeah, I don't think it's a problem to have them there... I mean, it's a little more data for Travis to be storing, but it's not that big, and it's nice being able to see where the errors came from.